extensions/assets-manager/AssetsManagerEx.h

```diff
@@ -67,16 +67,33 @@ class CC_EX_DLL AssetsManagerEx : public Ref
         FAIL_TO_UPDATE
     };
     
+    struct Config {
+        /** @brief  Function used to calculate the storage path of each asset
+         */
+        std::function<std::string(const std::string& key, const Manifest::Asset& asset)> getAssetStoragePath;
+
+        /** @brief  Hints for configuring downloader
+         */
+        network::DownloaderHints downloaderHints;
+
+        /** @brief  Default configuration for AssetsManagerEx
+         */
+        static const Config& defaultConfig();
```

Comment:
defaultConfig -> getDefaultConfig? because most of cocos2d-x function names starts with a verb.